client/coral-admin/src/routes/Community/components/People.js

```diff
@@ -130,7 +130,14 @@ class People extends React.Component {
                             {user.username}
                           </button>
                           <span className={styles.email}>
-                            {user.profiles.map(({ id }) => id)}
+                            {user.email
+                              ? user.email
+                              : user.profiles.map(({ id }, i) => {
```

Comment:
Is this the same as `user.profiles.map(p => p.id).join(', ')`?

Reply:
Typically, the `id` from the provider are quite long, hence why it probably shouldn't be displayed here, but is displayed on the `UserDetail` component.